lib/sort.js

```diff
@@ -36,6 +38,15 @@ module.exports = function sortDocs(comments, options) {
   var fixed = options.toc.filter(function (val) {
     return typeof val === 'object' && val.name;
   }).map(function (val) {
+    if (typeof val.file === 'string') {
+      var filename = path.join(process.cwd(), val.file);
```

Comment:
Thinking about this more, in a case like

    documentation build --config=some/directory/documentation.yml index.js

The paths in `documentation.yml` should be relative to `some/directory` rather than to `./`. In that case, we might want to [add the config path when we go through configParser](https://github.com/documentationjs/documentation/blob/43e01ce076fc943b2d52adab094b92b274893331/lib/commands/shared_options.js#L22-L24) ([docs for that on yargs](https://github.com/yargs/yargs#config)), so that `sort` can use it as a relative path if it's present.